Report/Chapters/Design.tex

```diff
@@ -52,7 +52,7 @@ \subsubsection{Menus}
 
 The primary menu (or `function menu') uses `drawers' for the categories of the functions. \index{primary menu} \index{function menu}
 This way it is easy to find functions even without an extensive knowledge of Haskell.
-In- and ontput blocks and the function definition block can be created using buttons at the bottom of the menu.
+In- and output blocks and the function definition block can be created using buttons at the bottom of the menu.
 This location is chosen to make them quickly available.
 The primary menu can be opened multiple times at once and moved around the screen so multiple users can work on a single program.
```

Comment:
Ans also in order to never be limitted by screen size (as opposed to when it is stuck on an edge)